src/main/resources/templates/instructor/base_instructor.twig

```diff
@@ -0,0 +1,25 @@
+{% extends '/templates/base.twig' %}
```

Comment:
I created a base instructor class which inherits from base.twig to minimize repeated html